revive.toml

```diff
@@ -0,0 +1,30 @@
+ignoreGeneratedHeader = false
+severity = "error"
+confidence = 0.8
+errorCode = -1
+warningCode = -1
+
+[rule.blank-imports]
+[rule.context-as-argument]
+[rule.context-keys-type]
+[rule.dot-imports]
+[rule.error-return]
+[rule.error-strings]
+[rule.error-naming]
+[rule.exported]
+[rule.if-return]
+[rule.increment-decrement]
+[rule.var-naming]
+[rule.var-declaration]
+[rule.package-comments]
+[rule.range]
+[rule.receiver-naming]
+[rule.time-naming]
+[rule.unexported-return]
+[rule.indent-error-flow]
+[rule.errorf]
+[rule.empty-block]
+[rule.superfluous-else]
+#[rule.unused-parameter]
```

Comment:
I am curious why is this rule commented?

Reply:
It created a ton of additional errors 😄 I wanted to limit scope of this change and just get rid of golint. We can consider enabling this check and others in the future

Reply:
Sounds good, so please don't forget to create an issue for this.

Reply:
Created #1107 and adjusted good first issue to say revive instead of golint